cmk/base/plugins/agent_based/checkpoint_connections.py

```diff
@@ -7,18 +7,66 @@
 # .1.3.6.1.2.1.1.1.0 Linux gateway1 2.6.18-92cp #1 SMP Tue Dec 4 21:44:22 IST 2012 i686
 # .1.3.6.1.4.1.2620.1.1.25.3.0 19190
 
-from .agent_based_api.v1 import register, SNMPTree
-from .utils import checkpoint
+from typing import NamedTuple
 
+from cmk.base.plugins.agent_based.agent_based_api.v1 import (
+    check_levels,
+    register,
+    Service,
+    SNMPTree,
+)
+from cmk.base.plugins.agent_based.agent_based_api.v1.type_defs import CheckResult, DiscoveryResult
+from cmk.base.plugins.agent_based.utils import checkpoint
+
+check_default_parameters = {"levels": (40000, 50000)}
```

Comment:
Please remove this variable and instead directly add the value to register.check_plugin(...)

Reply:
makes sense yes
about the render function: when I only used "lambda v: str(v)", I always got the advice that the use of lamba may not be necessary - can that be safely ignored?

Reply:
Actually, the warning is correct, this is something I keep overlooking. Of course,
`lambda v: str(v)`
is equivalent to
`str`
so you can use 
`render_func = str`